vignettes/development.qmd

```diff
@@ -0,0 +1,71 @@
+---
+title: "Feature List and Roadmap"
+vignette: >
+  %\VignetteIndexEntry{Feature List and Roadmap}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{quarto::html}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: "#>"
+---
+
+This document outlines the features of the `laminr` package and the roadmap for future development.
+
```

Comment:
I think it would be worthwhile to specify which features are implemented, planned, or won't be implemented any time soon. I found that figuring out which features we should implement is not trivial, though...